website/pages/docs/install/glossary.mdx

```diff
@@ -74,3 +74,333 @@ over the internet or wide area network.
+from different vendors in a single architecture.  This refers to the distribution 
+of compute resources, storage, and networking aspects across cloud environments.  
+A multi-cloud environment could be either all private cloud or all public cloud 
+or a combination of both. 
+
+See also [Hybrid Cloud](link to that glossary term).
+
+## Multi-cloud Networking	
+Multi-cloud Networking provides network configuration and management across 
+multiple cloud providers via APIs.
+
+## mutual Transport Layer Security (mTLS)	
+Mutual Transport Layer Security, also known as mTLS, is an authentication 
+mechanishm that ensures network traffic security in both directions between 
+a client and server. 
+
+## Network Middleware Automation	
+The process of publishing service changes to network middleware such as 
+load balancers and firewalls and automating network tasks is called Network 
+Middleware Automation.
+
+## Network security	
+Network security is the process of protecting data and network.  It consists 
+of a set of policies and practices that are designed to prevent and monitor 
+unauthorized access, misuse, modification, or denial of a computer network 
+and network-accessible resources.
+
+## Network traffic management	
+Network Traffic Management is the process of ensuring optimal network operation 
+by using a set of network monitoring tools.  Network traffic management also 
+focuses on traffic management techniques such as bandwidth monitoring, deep 
+packet inspection, and application based routing.
+
+## Network Visualization	
+Network Visualization is the process of visually displaying networks and 
+connected entitites in a "boxes and lines" kind of a diagram. 
+
+In the context of microservices architecture, visualization can provide a clear 
+picture of how services are connected to each other, the service-to-service 
+communication, and resource utilization of each service.
+
+## Observability	
+Observability is the process of logging, monitoring, and alerting on the 
+events of a deployment or an instance.
+
+## Elastic Scaling	
+Elastic Scaling is the ability to automatically add or remove compute or 
+networking resources based on the changes in application traffic patterns.
+
+## Platform as a Service	
+Platform-as-a-Service (PaaS) is a category of cloud computing that allows 
+users to develop, run, and manage applications without the complexity of 
+building and maintaining the infrastructure typically associated with 
+developing and launching the application. 
+
+## Reverse Proxy	
+A reverse proxy handles requests coming from outside, to the internal 
+network. Reverse Proxy provides a level of security that prevents the 
+external clients from having direct access to data on the corporate servers. 
+The reverse proxy is usually placed between the web server and the external 
+traffic.  
+See also [Forward Proxy](link to that glossary term).
+
+## Role-based Access Controls	"The act of restricting or provisioning access 
```

Comment:
Seems  like a missed formatting here.

Reply:
I meant to indicate that we should cross-link the terms to that specific spot on the page.  So when folks are looking at the definition of Reverse Proxy, we provide them a link to the definition of Forward Proxy on the same page.  Does that help?